src/state/nftMarket/hooks.ts

```diff
@@ -22,18 +21,23 @@ export const useFetchCollection = (collectionAddress: string) => {
   }, [dispatch, collectionAddress])
 }
 
-// Returns a function that fetches more NFTs when called and puts them into redux state.
-// Also returns loading flag and time of latest successful fetch
-export const useFetchByBunnyId = (bunnyId: string) => {
+// Returns a function that fetches more NFTs for specified bunny id
+// as well as updating existing PB NFTs in state
+// Note: PancakeBunny specific
+export const useFetchByBunnyIdAndUpdate = (bunnyId: string) => {
```

Comment:
Originally I rewrote this to account for other collections too, but the logic became too complex and resulted in more queries (even if user just sits on Dollop page it would update Squads, etc).
Its better to make a more maintainable hook for other collections, PB are quite different in terms of updating since they have multiple instances.